pom.xml

```diff
@@ -18,10 +18,15 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.23.1</version>
+        </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>4.13.2</version>
```

Comment:
What is the reason to change version?

Reply:
![image](https://github.com/mate-academy/jv-logger/assets/113580621/983b9271-4b76-422c-9706-9a5b9b4a400b)